sbt-plugin/src/main/scala/org/scalajs/sbtplugin/HTMLRunnerTemplate.scala

```diff
@@ -20,10 +20,9 @@ import org.scalajs.testadapter.TaskDefSerializers._
 /** Template for the HTML runner. */
 private[scalajs] object HTMLRunnerTemplate {
 
-  def render(baseURI: URI, title: String, sjsFile: URI, jsdepsFile: URI,
+  def render(baseURI: URI, title: String, jsFiles: Seq[URI],
```

Comment:
I haven't considered changing URIs here in this PR, because that can be done later without changing the public API. I'd rather limit changes in this PR to things that are necessary to the get the public API we need, and refine internals later.